dtls/Cargo.toml

```diff
@@ -0,0 +1,85 @@
+[package]
+name = "webrtc-dtls"
+version = "0.5.3"
+authors = ["Rain Liu <yuliu@webrtc.rs>"]
+edition = "2018"
+description = "A pure Rust implementation of DTLS"
+license = "MIT/Apache-2.0"
+documentation = "https://docs.rs/webrtc-dtls"
+homepage = "https://webrtc.rs"
+repository = "https://github.com/webrtc-rs/dtls"
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+
+[dependencies]
+util = { path = "../util", package = "webrtc-util", default-features = false, features = ["conn"] }
```

Comment:
I think the all workspace dependencies will need version numbers eventually, once you want to publish one of these.